pyproject.toml

```diff
@@ -44,19 +44,21 @@ ipython = "*"
 pylint = "*"
 pylint-django = "*"
 pylint-nautobot = "*"
+regex = "~2023.8.8"
```

Comment:
Is that needed somewhere?

Reply:
I had pinned this because of an error I was getting when running the CI, but this has been resolved by updating Nautobot's version, so I've removed it.
```  RuntimeError

  Unable to find installation candidates for regex (2024.5.15)

  at ~/.local/share/pypoetry/venv/lib/python3.11/site-packages/poetry/installation/chooser.py:74 in choose_for
       70│ 
       71│             links.append(link)
       72│ 
       73│         if not links:
    →  74│             raise RuntimeError(f"Unable to find installation candidates for {package}")
       75│ 
       76│         # Get the best link
       77│         chosen = max(links, key=lambda link: self._sort_key(package, link))
       78│ 

Cannot install regex.
```